packages/alfa-aria/src/role.ts

```diff
@@ -330,21 +329,31 @@ export namespace Role {
  *
  * @see https://w3c.github.io/aria/#conflict_resolution_presentation_none
  */
-const isAllowedPresentational: Predicate<Element> = (element) => {
-  if (element.tabIndex().isSome()) {
-    return false;
-  }
 
-  return Role.lookup("roletype").some((role) => {
+// An element is focusable if it has a tabindex and is not disabled.
+// Note that elements that are not rendered are still considered; however, these are not exposed in the accessibility
+// tree in the first place, so we should be fairly safe here.
```

Comment:
```suggestion
/**
 * An element is focusable if it has a tabindex and is not disabled. Note that 
 * elements that are not rendered are still considered; however, these are not 
 * exposed in the accessibility tree in the first place, so we should be fairly 
 * safe here.
 */
```

Reply:
Top level declarations should use docblocks to enable intellisense and comments should be wrapped at 80 columns.